packages/@react-spectrum/inlinealert/intl/bg-BG.json

```diff
@@ -0,0 +1,6 @@
+{
+  "negative": "Грешка",
+  "info": "Информация",
```

Comment:
the only overlap between the two packages was the info string, we should remove the others

Reply:
The change was essentially:

```diff
{
  "neutral": "Neutral",
-  "info": "Informative",
+  "info": "Information",
-  "positive": "Positive",
+  "positive": "Success",
-  "notice": "Notice",
+  "notice": "Warning",
-  "negative": "Negative"
+  "negative": "Error"
}
```

These are for alt text for the icon so IMO the updates are fine. Even info didn't overlap perfectly, so I can remove them all if we want to do that and just wait for translations.

Reply:
And now that I think about it, we can remove the neutral string since there isn't an icon for the neutral variant.

Reply:
Prob move `iconAlt` into the `if (variant in ICONS) {` check in Inline alert